ray-operator/config/samples/ray-cluster.heterogeneous.yaml

```diff
@@ -152,8 +150,8 @@ spec:
     # if worker pods need to be removed, we decrement the replicas, and populate the podsToDelete list
     # the operator will remove pods from the list until the number of replicas is satisfied
     # when a pod is confirmed to be deleted, its name will be removed from the list below
-    scaleStrategy:
-      workersToDelete:
+#    scaleStrategy:
+#      workersToDelete:
```

Comment:
![ray-cluster-hetero-error](https://user-images.githubusercontent.com/37469330/135425181-36d64875-354c-4f91-99ee-fc4e5dc277e8.png)

When deploying this file in raw, above error was occured. So I deleted this field, same as above